docs/docs/micromatch-wrapper.md

```diff
@@ -0,0 +1,52 @@
+#### Overview
+[Micromatch](https://github.com/micromatch/micromatch) support various matching features, such as Glob pattern, Advanced globbing.
+
+Micromatch is a nodejs application, and MBT is a go application. In order to enable MBT to use the functionalities provided by micromatch, from MBT v1.2.25, we create [micromath wrapper](https://github.com/SAP/cloud-mta-build-tool/tree/master/micromatch) to package it.

```

Comment:
Micromatch is a Node.js application, while MBT is a Go application. The [micromatch wrapper](https://github.com/SAP/cloud-mta-build-tool/tree/master/micromatch) is a package that enables MBT to use the functionalities provided by micromatch.

Reply:
link is correct.

micromatch-wrapper.md is new added file in this PR, in feature branch mbt-upgrade-build-parameter-ignore, the link is : https://github.com/SAP/cloud-mta-build-tool/blob/mbt-upgrade-build-parameter-ignore/docs/docs/micromatch-wrapper.md

The link https://github.com/SAP/cloud-mta-build-tool/tree/master/docs/docs/micromatch-wrapper.md will be work after this PR merged into master